desktop/cmp/panel/impl/dragger/DraggerModel.js

```diff
@@ -63,19 +64,19 @@ export class DraggerModel {
 
         const {clientX, clientY} = this.parseEventPositions(e);
         this.resizeState = {startX: clientX, startY: clientY};
-        this.startSize = panelModel.size;
+        this.startSize = panel[vertical ? 'offsetHeight' : 'offsetWidth'];
         this.panelParent = panel.parentElement;
         panelModel.setIsResizing(true);
 
-        if (!panelModel.resizeWhileDragging) {
+        if (!resizeWhileDragging) {
             this.dragBar = this.getDraggableSplitter(dragger);
             this.panelParent.appendChild(this.dragBar);
             this.diff = 0;
         }
 
         // We will use whichever is smaller - the calculated available size, or the configured max size
         const calcMaxSize = this.startSize + this.getSiblingAvailSize();
-        this.maxSize = panelModel.maxSize ? Math.min(panelModel.maxSize, calcMaxSize) : calcMaxSize;
+        this.maxSize = maxSize ? Math.min(maxSize, calcMaxSize) : calcMaxSize;
```

Comment:
What happens here if PanelModel.maxSize is a percentage value? It looks like the rest of DraggerModel expects a pixel value for `this.maxSize`

Reply:
Disregard, shouldn't be a problem if `PanelModel.maxSize` is still always a px int